HtmlMesh/src/fit-strategy.ts

```diff
@@ -88,9 +88,16 @@ const FitStrategyNone: FitStrategyType = {
   }
 }
 
-export const FitStrategy = {
-  CONTAIN: FitStrategyContain,
-  COVER: FitStrategyCover,
-  STRETCH: FitStrategyStretch,
-  NONE: FitStrategyNone
+export enum FitStrategy {
+    CONTAIN = 3,
+    COVER = 2,
+    STRETCH = 1,
+    NONE = 0
+}
+
+export const FitStrategyMap = {
```

Comment:
Can you explain a little why this was necessary, i.e. what issue were you running into?  The current implementation is consistent with a pattern that is used elsewhere in Babylon, e.g. `RenderSortOrderCompare` functions, so that is why that pattern was chosen.

Reply:
Yes, it should have been exposed, that was an oversight.  However, I'm not a fan of changing fitStrategy to be an enum.  I would want a developer to be able to use a custom fit strategy that isn't one of the exported strategies.  Also, this creates a breaking change in the constructor, because the implicit type of fit strategy has changed from a FitStrategyType to an enum value.  Could you please keep the export, but remove the enum so a developer can specify a FitStrategy that isn't one of the exported types?